includes/class-migration.php

```diff
@@ -1051,4 +1055,28 @@ public static function sync_jetpack_following_meta() {
 			\do_action( 'added_post_meta', ...$meta );
 		}
 	}
+
+	/**
+	 * Cleanup inbox items for shared inbox migration.
```

Comment:
```suggestion
	 * Clean up inbox items for shared inbox migration.
```
This is similar to setup and set up, where cleanup is the noun and clean up the verb.